warp10/src/main/java/io/warp10/script/functions/MANAGERONOFF.java

```diff
@@ -19,35 +19,36 @@
 import io.warp10.WarpManager;
 import io.warp10.script.NamedWarpScriptFunction;
 import io.warp10.script.WarpScriptStackFunction;
+import io.warp10.warp.sdk.Capabilities;
 import io.warp10.script.WarpScriptException;
 import io.warp10.script.WarpScriptStack;
 
 public class MANAGERONOFF extends NamedWarpScriptFunction implements WarpScriptStackFunction {
-  
+
   private final boolean on;
   private final String attr;
-  
+
   public MANAGERONOFF(String name, String attr, boolean on) {
     super(name);
     this.on = on;
     this.attr = attr;
   }
-  
+
   @Override
   public Object apply(WarpScriptStack stack) throws WarpScriptException {
     Object top = stack.pop();
```

Comment:
As the manager secret is not needed anymore, this line can be removed.
The documentation of the implemented WarpScript functions should be updated accordingly.